src/routes/safe/components/Apps/components/SecurityFeedbackModal/__tests__/index.test.tsx

```diff
@@ -94,4 +97,14 @@ describe('<SecurityFeedbackModal />', () => {
 
     expect(baseProps.onCancel).toHaveBeenCalledTimes(1)
   })
+
+  it('should prompt the user for permissions', async () => {
+    // @ts-ignore
```

Comment:
Why the ts-ignore?

Reply:
Removed. Something that remained from the first attempt to test the component